source/administration/changelog.md

```diff
@@ -51,13 +50,13 @@ Added significant improvements for large organizations running on a single serve
 ### Improvements
 
 #### iOS Apps
-- Channel and account settings, and channel header are now full screen modals for better visibility
+- Channel settings, account settings, and channel header render are now full screen modals for better visibility
```

Comment:
Sentence seems weird, did you mean `Channel settings, account settings, and channel header now render full screen for better visibility`